src/swap/SwapScreen.test.tsx

```diff
@@ -26,8 +34,14 @@ jest.mock('react-native-localize', () => ({
 }))
 
 const now = Date.now()
-
-const renderScreen = ({ celoBalance = '10', cUSDBalance = '20.456' }) => {
+const emptyProps = {} as NativeStackScreenProps<StackParamList, Screens.SwapScreenWithBack>
+
+const renderScreen = ({
+  celoBalance = '10',
+  cUSDBalance = '20.456',
+  showDrawerTopNav = true,
+  props = emptyProps,
```

Comment:
nit: you can just default this to undefined since its not a required prop right?

Reply:
I can't because the type of `props` is inferred from the default value. If `props` is default `undefined`, I can't later call the `renderScreen` function with any other value for `props`

Reply:
you can define types explicitly instead of inferred types, but don't feel too strongly on this